celer/dropin_sklearn.py

```diff
@@ -130,12 +135,15 @@ def __init__(self, alpha=1., max_iter=100, gap_freq=10,
 
     def path(self, X, y, alphas, **kwargs):
         """Compute Lasso path with Celer."""
+        if not self.warm_start:
```

Comment:
@MalcolmWolff do we need this? I think it is done diffrently in sklearn lasso_path